src/events/interfaces/event.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface Event {
+  id: string;
+  event: string;
+  created_at: string;
+  data: Record<string, unknown>;
+}
```

Comment:
I think we can be more specific here, like we are [with webhook types](https://github.com/workos/workos-node/blob/main/src/webhooks/interfaces/webhook.interface.ts).